### DISCUSSION TITLE
MacOS Sonoma has a switch for pointer acceleration

### DISCUSSION THREAD

**Discussion by ziyanTOP** [linearmouse/linearmouse#493] [Q&A] [answered]
Is this configuration same as the config which linearMouse reads from or writes to?

![image](https://github.com/linearmouse/linearmouse/assets/49580493/9d4170cd-67c0-40b1-b8b0-6e28fff07347)

![image](https://github.com/linearmouse/linearmouse/assets/49580493/48a5623f-3366-4ea6-883a-2a2e711e6351)

**Comment by lujjjh** [comment, marked as answer]
Edit: I've moved to the new API in #496. So the answer is yes, but only for macOS Sonoma, and you can still configure this setting for specific device or app.
